src/utils/rates.js

```diff
@@ -36,7 +36,7 @@ export const getUsdToFiatRate = (rates: RatesByAssetAddress, currency: Currency)
   if (currency === USD) return 1;
 
   // will select native asset rates to calculate between
-  const nativeAssetRates = rates[ethers.constants.AddressZero];
+  const nativeAssetRates = rates[valueAsKey(ethers.constants.AddressZero)];
```

Comment:
Can we have zero address re-exported from e.g. assetConstants. Importing ethers every time is a bit tedious.